stripe-core/src/main/java/com/stripe/android/core/ApiVersion.kt

```diff
@@ -9,29 +11,30 @@ package com.stripe.android
  * See [https://stripe.com/docs/upgrades](https://stripe.com/docs/upgrades) for latest
  * API changes.
  */
-internal data class ApiVersion internal constructor(
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+data class ApiVersion internal constructor(
     internal val version: String,
-    internal val betas: Set<StripeApiBeta> = emptySet()
+    internal val betaCodes: Set<String> = emptySet()
```

Comment:
An alternative to just having a String would be to use a value class that any module can define: 

```kotlin
// :stripe-core
value class StripeApiBeta(private val value: String)

// :payments
val apiVersion = ApiVersion(StripeApiBeta("we_chat_pay=v1"))
```

Reply:
Thanks for the tip, just tried this, unfortunately `StripeApiBeta` is a public API that's exposed through [Stripe.kt](https://github.com/stripe/stripe-android/blob/e72afad4822f05e1fe817d3d7f1b721fb82f13a6/payments-core/src/main/java/com/stripe/android/Stripe.kt#L97), changing it would introduce some backward incompatible changes, will wait this for the next major version bump